clients/include/testing_matmul.hpp

```diff
@@ -90,11 +96,17 @@ void epilogue_func(int64_t m,
                    To*     out,
                    Tc*     out_raw,
                    Tc*     e,
+                   bool    doScaleD,
```

Comment:
doScaleD and doScaleE are unneccesary. scaleD and scaleE would be set as 1 in disabling scaleD or scaleE cases.

Reply:
fixed